node/src/actors/session/handlers.rs

```diff
@@ -572,7 +593,7 @@ fn handshake_version(session: &mut Session, sender_address: &Address) -> Vec<Wit
             session.magic_number,
             session.server_addr,
             session.remote_addr,
-            0,
+            session.current_epoch,
```

Comment:
But current_epoch is not used anymore, right? We use the timestamp instead. Can we remove the `current_epoch` field from the `Version` message?

Reply:
we always had a useless current_epoch in the Version :P

Reply:
New issue?

Reply:
created #888 issue for that